docs/contents/cloud-bigtable.json

```diff
@@ -0,0 +1,25 @@
+{
+    "title": "Bigtable",
+    "pattern": "bigtable\/\\w{1,}",
+    "services": [{
+        "title": "BigtableClient",
+        "type": "bigtable/bigtableclient"
```

Comment:
Apologies for all the back and forth on this one, it is confusing because of the way we document GAPICs differently from the rest of the manual veneer clients. This needs to be `bigtable/readme`.

Reply:
Done.